dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/restmetrics/RestMetricsTest.java

```diff
@@ -241,6 +285,46 @@ private void runCheckMonitorStats(int exprc, int index) throws IOException {
         }
     }
 
+    private void runHeadMethod(int index, int exprc, String requestUri, String testOut) throws IOException {
```

Comment:
Could you combine runHeadMethod with runGetMethod (and maybe other methods) by adding another parameter (i.e. `httpMethod`) and then pass "GET" or "HEAD" to the new method?  That might make this easier to maintain.

Reply:
Unfortunately the runGetMethod and run HeadMethod have subtle differences,  but this comment is valid in general.   I will look for ways to combine methods.